spec/controllers/sign_up/completions_controller_spec.rb

```diff
@@ -222,11 +218,12 @@
         user = create(:user)
         sp = create(:service_provider, issuer: 'https://awesome')
         stub_sign_in(user)
-        subject.session[:sp] = { issuer: sp.issuer,
-                                 ial2: false,
-                                 acr_values: Saml::Idp::Constants::IAL1_AUTHN_CONTEXT_CLASSREF,
-                                 requested_attributes: [:email],
-                                 request_url: 'http://localhost:3000' }
+        subject.session[:sp] = {
+          issuer: sp.issuer,
+          acr_values: Saml::Idp::Constants::IAL1_AUTHN_CONTEXT_CLASSREF,
+          requested_attributes: [:email],
+          request_url: 'http://localhost:3000',
+          }
```

Comment:
style nit, I would unindent here
```suggestion
          request_url: 'http://localhost:3000',
        }
```